packages/theming/apple-theme/FluentUIReactNativeAppleTheme.podspec

```diff
@@ -0,0 +1,25 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = 'FluentUI-React-Native-Apple-Theme'
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+
+  s.source           = { :git => "https://github.com/microsoft/fluentui-react-native.git", :tag => "#{s.version}" }
+  s.swift_version    = "5"
+
+  # s.ios.deployment_target = "11.0"
```

Comment:
Is the plan here that when we get iOS support we uncomment this?

Reply:
Yep. I'm also happy to just remove this, didn't notice this was here

Reply:
Probably worth removing for now or adding a TODO with a bug number tracking adding iOS support